esphome/components/mopeka_pro_check/sensor.py

```diff
@@ -89,6 +97,26 @@ def small_distance(value):
                 device_class=DEVICE_CLASS_BATTERY,
                 state_class=STATE_CLASS_MEASUREMENT,
             ),
+            cv.Optional(CONF_SIGNAL_STRENGTH): sensor.sensor_schema(
```

Comment:
Since this is labelled as signal quality everywhere, the config name should match
```suggestion
            cv.Optional(CONF_SIGNAL_QUALITY): sensor.sensor_schema(
```